app/controllers/util/FormField.scala

```diff
@@ -122,18 +133,33 @@ object RadioButtonGroupField {
   }
 }
 
+object MultiSelectField {
+
+  def validate(slectedOptions: Seq[String], multiSelectField: MultiSelectField): Option[String] =
+    slectedOptions match {
+      case Nil                                                                      => Some(dropdownOptionNotSelectedError.format(multiSelectField.fieldName))
+      case values if !values.forall(multiSelectField.options.map(_.value).contains) => Some(invalidDropdownOptionSelectedError.format(values.mkString(", ")))
+      case _                                                                        => None
+    }
+
+  def update(multiSelectField: MultiSelectField, selectedValues: Seq[String]): MultiSelectField = {
+    val selectedOptions: List[InputNameAndValue] = multiSelectField.options.filter(v => selectedValues.contains(v.value)).toList
+    multiSelectField.copy(selectedOption = Some(selectedOptions))
+  }
+}
+
 object DropdownField {
 
-  def validate(option: String, dropdownField: DropdownField): Option[String] =
-    option match {
-      case ""                                                       => Some(dropdownOptionNotSelectedError.format(dropdownField.fieldName))
-      case value if !dropdownField.options.exists(_.value == value) => Some(invalidDropdownOptionSelectedError.format(value))
-      case _                                                        => None
+  def validate(selectedOption: Option[String], dropdownField: DropdownField): Option[String] =
+    selectedOption match {
+      case None                                                           => Some(dropdownOptionNotSelectedError.format(dropdownField.fieldName))
+      case Some(value) if !dropdownField.options.exists(_.value == value) => Some(invalidDropdownOptionSelectedError.format(value))
+      case _                                                              => None
     }
 
-  def update(dropdownField: DropdownField, value: String): DropdownField = {
-    val optionSelected: Option[InputNameAndValue] = if (value.isEmpty) None else Some(InputNameAndValue(value, value))
-    dropdownField.copy(selectedOption = optionSelected)
+  def update(multiSelectField: DropdownField, selectedValue: Option[String]): DropdownField = {
```

Comment:
Think the `multiSelectField` parameter should still be named: `dropdownField`, as this is a method for the `DropdownField` object.